apps/dotcms-block-editor/src/app/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -33,7 +33,7 @@ function toTitleCase(str) {
 })
 export class DotBlockEditorComponent implements OnInit {
     editor: Editor;
-
+    @Input() allowedContentTypes = '';
```

Comment:
Do we need this if we're getting this from the HTML?

Reply:
yes, was changed here as input.